tsdb/chunkenc/histogram.go

```diff
@@ -265,14 +265,15 @@ func (a *HistogramAppender) Append(int64, float64) {
 func (a *HistogramAppender) appendable(h *histogram.Histogram) (
 	positiveInserts, negativeInserts []Insert,
 	backwardPositiveInserts, backwardNegativeInserts []Insert,
-	okToAppend, counterReset bool,
+	okToAppend bool, counterResetHeader CounterResetHeader,
```

Comment:
With the new return type, the doc comment above needs an update, ideally explaining that `UnknownCounterReset` is returned in certain cases to avoid the work of a full counter reset detection.

And let's call the variable something like `counterReset` or `counterResetHint`. We are using the `CounterResetHeader` type, but it's not really a header what we are returning.

Reply:
Done